documentation/src/css/custom.css

```
@@ -418,3 +418,29 @@
+ * Infima re-declares --ifm-h2/h3-font-size on the child selectors
+ * (.markdown > h2, .markdown > h3) and bumps the first h1 to 3rem, so we must
+ * override at those same selectors — a plain `.markdown { ... }` var override
+ * is out-specified by them and has no effect.
+ */
+.markdown h1:first-child {
+  --ifm-h1-font-size: 1.75rem;
+}
+
+.markdown > h2 {
+  --ifm-h2-font-size: 1.375rem;
+  font-weight: 600;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)